pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -226,6 +228,89 @@ var _ = Describe("podNIC", func() {
 			storedState: ptr(cache.PodIfaceNetworkingPrepared),
 		}),
 	)
+	Context("state transition", func() {
+		When("state is PodIfaceNetworkingUnprepared", func() {
+			var (
+				podnic *podNIC
+			)
+			BeforeEach(func() {
+				var err error
+				vmi := newVMIBridgeInterface("testnamespace", "testVmName")
+				podnic, err = newPhase1PodNICWithMocks(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				pid := 1
+				podnic.launcherPID = &pid
+				podnic.setState(cache.PodIfaceNetworkingUnprepared)
+				address1 := &net.IPNet{IP: net.IPv4(1, 2, 3, 4)}
+				address2 := &net.IPNet{IP: net.IPv4(169, 254, 0, 0)}
+				fakeAddr1 := netlink.Addr{IPNet: address1}
+				fakeAddr2 := netlink.Addr{IPNet: address2}
```

Comment:
Can't you inline these ? AFAIU you're not using `address1` or `address2` anywhere else. That would enable you to not call variables `fakeAddr` - which makes me wonder what is a real address.

Reply:
Also removing "fake" prefix